cookbooks/aws-parallelcluster-platform/libraries/write_chef_error_handler.rb

```diff
@@ -21,56 +21,51 @@ def report
       extend Chef::Mixin::ShellOut
       error_file = node['cluster']['bootstrap_error_path']
 
-      # to avoid overwriting the error message from other mechanisms, such as the deprecated BYOS handler
-      # if the error file already exists we don't take any additional action here
-      unless File.exist?(error_file)
-        message_error = 'Failed to run chef recipe.'
-        message_logs_to_check = \
-          'Please check /var/log/chef-client.log in the head node, or check the chef-client.log in CloudWatch logs.'
-        message_troubleshooting_link = 'Please refer to'\
-          ' https://docs.aws.amazon.com/parallelcluster/latest/ug/troubleshooting-v3.html'\
-          ' for more details.'
+      message_error = 'Failed to run chef recipe.'
```

Comment:
A cluster can perform update for multiple times. We don't want to surface the same error messages when next time the update succeeded. We can consider a way to rename the file to a backup fole on the CLI side after it get surface or update failed.